### PR TITLE
PAYARA-2998 Document Add rotation on file fize options on Access logging

### PR DESCRIPTION
**Pull request by MeroRai**

### CODE REVIEW

File: documentation/payara-server/logging/acsess-logging-max-file-size.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[[access-logging-max-file-size]]
+= Access Logging Max File Size
+
+_Since version 5.183_
+
+Both Payara Server and GlassFish 4.0 Open Source Edition provides different ways
```
Comment (OndrejM) [inline documentation/payara-server/logging/acsess-logging-max-file-size.adoc]: We don't need to mention the version of GlassFish here, just "GlassFish Open Source Edition"

Comment (arjantijms) [reply documentation/payara-server/logging/acsess-logging-max-file-size.adoc]: "acsess-logging-max-file" shouldn't that be "access-logging-max-file"?

File: documentation/payara-server/logging/access-logging-max-file-size.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+== Using the Admin Console
+
+To configure `Max File Size` option using Admin Console:
+
+image:/images/logging/max_file_size.png[Max File Size]
+
+[[using-asadmin-utility]]
+== Using an `asadmin` command
+
+Use the following command to set maximum size for an access log files:
```
[the remark this answers is not in the document]
Comment (OndrejM) [reply documentation/payara-server/logging/access-logging-max-file-size.adoc]: I'll fix this later.

File: SUMMARY.adoc [hunk 3]
```diff
@@ -36,11 +36,12 @@
 ** link:documentation/payara-server/jdbc/jdbc-connection-validation.adoc[JDBC]
 ** link:documentation/payara-server/h2/h2.adoc[H2 Database]
 ** link:documentation/payara-server/logging/logging.adoc[Logging]
+*** link:documentation/payara-server/logging/acsess-logging-max-file-size.adoc[Access Logging Max File Size]
```
Comment (OndrejM) [inline SUMMARY.adoc]: should be "access" not "acsess", the file isn't linked

File: documentation/payara-server/logging/access-logging-max-file-size.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (MeroRai) [reply documentation/payara-server/logging/access-logging-max-file-size.adoc]: @OndrejM I have fixed it

[the remark this answers is not in the document]
Comment (MeroRai) [reply documentation/payara-server/logging/access-logging-max-file-size.adoc]: @OndrejM I fixed it

**Pull request merged**